samples/dataQuery/command.js

```diff
@@ -0,0 +1,58 @@
+const Superagent = require('superagent');
+const mysql = require('promise-mysql');
+/**
+* This is the implementation of your Fusebot command.
+* You can run it from Slack with `/fusebot run-query ...`.
+* Fusebot is powered by the Fusebit integration platform, https://fusebit.io/
+* 
+* Docs and samples are at https://github.com/fusebit/fusebot
+* You can talk to a fellow developer on Slack at https://fusebit.io/contact
+* Or tell us what you think with `/fusebot feedback {your-comments}` 
+*
+* Now let's build something already. 
+
+* @param ctx {FusebotContext}
+*/
+module.exports = async (ctx) => {
+    if (ctx.body.args.length === 0 || ctx.body.args[0] === "help") {
+        return help(ctx)
+    }
+    if (!ctx.configuration.HOST || !ctx.configuration.PORT || !ctx.configuration.USERNAME || !ctx.configuration.PASSWORD) {
+        return ctx.client.sendEphemeral("Unable to detect SQL connection settings from configuration. For more information, please check https://github.com/fusebit/fusebot/tree/main/samples/dataQuery")
+    }
+    const sqlStatement = ctx.body.args.join(" ")
+    try {
+        const conn = await mysql.createConnection({
+            host: ctx.configuration.HOST,
+            port: ctx.configuration.PORT,
+            user: ctx.configuration.USERNAME,
+            password: ctx.configuration.PASSWORD
+        })
+        const results = await conn.query(sqlStatement)
+        let totalRows = 0;
+        for (const row of results) {
+            if (totalRows >= 20) {
+                await ctx.client.send("Please limit your SQL query to 20 rows.")
+            }
+            await ctx.client.send(JSON.stringify(row))
+            totalRows++
+        }
```

Comment:
Not sure I get this logic. It will still display ALL the rows in the result, but add that extra "please limit.." text after the 20th. I was suggesting ew only display 20. Did you maybe forgot a `return`?

Reply:
The way I'd do it BTW is something like
```
const length = results.length;
results.splice(20);
await ctx.client.send(JSON.stringify(results));
if (length > 20) {
  await ctx.client.send(`...truncated. Query returned ${length} results, showing only first 20.`);
}
```

This will make only 1 or 2 HTTP calls instead of 20. 
}

Reply:
Done